### PR TITLE
RC: resilient apps tip

### PR DESCRIPTION
**Pull request by kaitlynmichael**

### CODE REVIEW

File: content/operate/rc/subscriptions/maintenance/_index.md [hunk 1]
```diff
@@ -41,11 +41,13 @@ This includes, but is not limited to:
...
 
-Your application may also disconnect from your database for a few seconds. Most Redis clients are set to refresh their DNS address when they reconnect to the database, and you will not be required to perform any further action. If you encounter connectivity problems for more than a minute during maintenance, please refresh your DNS entries. 
+Your application may also disconnect from your database for a few seconds. Most Redis clients are set to refresh their DNS address when they reconnect to the database, and you will not be required to perform any further action. If you encounter connectivity problems for more than a minute during maintenance, please refresh your DNS entries.
 
-See [Resilient applications with Redis Cloud]({{< relref "/operate/rc/resilient-apps" >}}) to learn how to develop highly resilient applications that can reconnect to your database quickly after maintenance.
+{{<tip>}}
+To make sure your applications reconnect to your database quickly after maintenance, see [Resilient applications with Redis Cloud]({{< relref "/operate/rc/resilient-apps" >}}).
```
Comment (mich-elle-luna) [inline content/operate/rc/subscriptions/maintenance/_index.md:49]:
```suggestion
To make sure your applications reconnect to your database quickly after maintenance, see [Develop highly available and resilient apps with Redis Cloud]({{< relref "/operate/rc/resilient-apps" >}}).
```

**Review by mich-elle-luna — APPROVED** [review]
One change to make the title match, otherwise looks good. Thank you!

File: content/operate/rc/subscriptions/maintenance/_index.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ During maintenance, your database will be operational, but you may notice some l
 Your application may also disconnect from your database for a few seconds. Most Redis clients are set to refresh their DNS address when they reconnect to the database, and you will not be required to perform any further action. If you encounter connectivity problems for more than a minute during maintenance, please refresh your DNS entries.
 
 {{<tip>}}
-To make sure your applications reconnect to your database quickly after maintenance, see [Resilient applications with Redis Cloud]({{< relref "/operate/rc/resilient-apps" >}}).
+To make sure your applications reconnect to your database quickly after maintenance, see [Develop highly available and resilient apps with Redis Cloud]({{< relref "/operate/rc/resilient-apps" >}}).
```
Comment (NoamSternRedis) [inline content/operate/rc/subscriptions/maintenance/_index.md:49]: Let's drop the word "quickly". It's problematic in our world and depends on the user. 
I would say something like "To make sure your applications are set to reconnect after maintenance activity"

**Review by NoamSternRedis — CHANGES_REQUESTED** [review]

File: content/operate/rc/subscriptions/maintenance/_index.md [hunk 3]
```diff
@@ -41,11 +41,13 @@ This includes, but is not limited to:
...
 
-Your application may also disconnect from your database for a few seconds. Most Redis clients are set to refresh their DNS address when they reconnect to the database, and you will not be required to perform any further action. If you encounter connectivity problems for more than a minute during maintenance, please refresh your DNS entries. 
+Your application may also disconnect from your database for a few seconds. Most Redis clients are set to refresh their DNS address when they reconnect to the database, and you will not be required to perform any further action. If you encounter connectivity problems for more than a minute during maintenance, please refresh your DNS entries.
 
-See [Resilient applications with Redis Cloud]({{< relref "/operate/rc/resilient-apps" >}}) to learn how to develop highly resilient applications that can reconnect to your database quickly after maintenance.
+{{<tip>}}
+To make sure your applications reconnect to your database quickly after maintenance, see [Develop highly available and resilient apps with Redis Cloud]({{< relref "/operate/rc/resilient-apps" >}}).
```
Comment (kaitlynmichael) [inline content/operate/rc/subscriptions/maintenance/_index.md:49]:
```suggestion
To make sure your applications are set to reconnect after maintenance activity, see [Develop highly available and resilient apps with Redis Cloud]({{< relref "/operate/rc/resilient-apps" >}}).
```

**Review by NoamSternRedis — APPROVED** [review]

**Pull request merged**